webplatformcompat/view_serializers.py

```diff
@@ -21,7 +21,14 @@
 from .serializers import (
     BrowserSerializer, FieldMapMixin, FeatureSerializer, MaturitySerializer,
     SectionSerializer, SpecificationSerializer, SupportSerializer,
-    VersionSerializer, omit_some)
+    VersionSerializer, FieldsExtraMixin)
+
+
+def omit_some(source_list, *omitted):
+    """Return a list with some items omitted"""
+    for item in omitted:
+        assert item in source_list, '%r not in %r' % (item, source_list)
```

Comment:
``omit_some`` is only used in this module, and only run when the code is initially parsed.  Here's a typical usage:

```
class ViewBrowserSerializer(BrowserSerializer):
     class Meta(BrowserSerializer.Meta):
         fields = omit_some(BrowserSerializer.Meta.fields, 'versions')
```

The equivalent without ``omit_some`` would be:

```
class ViewBrowserSerializer(BrowserSerializer):
     class Meta(BrowserSerializer.Meta):
         fields = (
            'id', 'slug', 'name', 'note', 'history_current', 'history')
```

which I think makes it harder to see what is left out.

I'm not sure why I'm using ``%r`` instead of ``%s``. I'll try out some combinations and see what happens with unicode, etc. If this assert fails, it will fail during initial server startup, not during runtime.